modules/persistent-storage-csi-manila-dynamic-provisioning.adoc

```diff
@@ -65,8 +68,8 @@ spec:
   storageClassName: csi-manila-gold <2>
 ----
 +
-<1> Use RWX if you want the persistent volume (PV) that fulfills this PVC to be mounted to multiple Pods on multiple nodes in the cluster. Use ReadWriteOnce (RWO) mode to prevent additional Pods from being dynamically provisioned.
-<2> The name of the storage class that provisions the storage back end. Manila StorageClasses are provisioned by the Operator and have the `csi-manila-` prefix.
+<1> Use RWX if you want the persistent volume (PV) that fulfills this PVC to be mounted to multiple pods on multiple nodes in the cluster. Use ReadWriteOnce (RWO) mode to prevent additional pods from being dynamically provisioned.
```

Comment:
@bobfuru There is some description of RWO too.

Reply:
Removed RWO sentence here.